### PR TITLE
update color schema for metric combo

### PR DESCRIPTION
**Pull request by apryka**
## How should this be tested?

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [ ] Does the code compile without warnings (check shell, console)?
- [ ] Do all tests pass?
- [ ] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
- [ ] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
- [ ] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [ ] If the feature saves data to a database, can you confirm the data is indeed created in the database?

### CODE REVIEW

File: lib/libraries/default/metric-combo.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ export default class MetricCombo {
   render() {
     const {
       container,
-      colors,
```
Comment (adamkasprowicz) [inline lib/libraries/default/metric-combo.js]:
```
colors = ['#c3c4cc'],
```

File: lib/libraries/default/metric-combo.js [hunk 2]
```diff
@@ -7,12 +7,12 @@ export default class MetricCombo {
...
       metricResults,
       title,
       tooltip,
     } = this.config;
 
+    const colors = ['#c3c4cc'];
```
Comment (adamkasprowicz) [inline lib/libraries/default/metric-combo.js]: delete this one

File: lib/libraries/default/metric-combo.js (hunk 1 above)
Comment (apryka) [reply lib/libraries/default/metric-combo.js]: colors are always present right now, inherited from Dataviz default options
